frontend/hub/collections/CollectionPage/CollectionContents.tsx

```diff
@@ -129,7 +130,7 @@ export function CollectionContents() {
           <RenderCommunityWarningMessage />
         )}
       </PageSection>
-    </>
+    </Scrollable>
```

Comment:
We could add some margin to README.

<img width="1178" alt="image" src="https://github.com/ansible/ansible-ui/assets/9053044/69b7212d-cedd-41ab-8530-f15ef6d1a909">



Old UI

![image](https://github.com/ansible/ansible-ui/assets/9053044/8c564021-7340-488e-aca5-092659399a30)